platform/podjs/src/manifest-parser/src/tests/__snapshots__/manifest.test.ts.snap

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Again, please consider to simply delete this. It can be tested directly, if needed.